crates/devolutions-pedm/src/api/log.rs

```diff
@@ -1,11 +1,30 @@
 use crate::api::{AppState, Db, NamedPipeConnectInfo};
 use crate::error::Error;
-use crate::log::{JitElevationLogPage, JitElevationLogQueryOptions};
+use crate::log::{JitElevationLogPage, JitElevationLogQueryOptions, JitElevationLogRow};
 use aide::axum::ApiRouter;
 use aide::NoApi;
-use axum::extract::State;
+use axum::extract::{Path, State};
 use axum::{Extension, Json};
 
+use super::policy::PathIntIdPath;
+
+async fn get_jit_elevation_log_id(
```

Comment:
naming: I think it’s easier to understand if a `by` is inserted
```suggestion
async fn get_jit_elevation_log_by_id(
```

Reply:
Okay! I don’t think it’s worth following this pattern, but I’ll rename in a separate PR (if I don’t forget :joy:)